src/rpc/blockchain.cpp

```diff
@@ -926,6 +931,7 @@ static RPCHelpMan getblock()
                         {RPCResult::Type::OBJ, "", "",
                         {
                             {RPCResult::Type::ELISION, "", "The transactions in the format of the getrawtransaction RPC. Different from verbosity = 1 \"tx\" result"},
+                            {RPCResult::Type::NUM, "fee", "The transaction fee in " + CURRENCY_UNIT + ", omitted if efficient calculation isn't possible (due to unavailable block undo data)"},
```

Comment:
I think this would be fine as just "omitted if block undo data is not available". I don't think users would care about or understand what you mean by 'efficient calculation'.